packages/blocks/azure/test/index.test.ts

```diff
@@ -1,3 +1,13 @@
+const openOpsMock = {
```

Comment:
for reasons beyond my understanding, the test now fails if its not mocked, sometimes i dont understand jest but i guess somewhere the common mock is not properly dealt with? who knows